packages/airnode-utilities/src/constants.ts

```diff
@@ -7,3 +7,7 @@ export const PRIORITY_FEE_IN_WEI = 3_120_000_000;
 export const BASE_FEE_MULTIPLIER = 2;
 // By default there is no gas price multiplier
 export const GAS_PRICE_MULTIPLIER = 1;
+export const RANDOM_BACKOFF_MIN_MS = 0;
+export const RANDOM_BACKOFF_MAX_MS = 2_500;
+export const GAS_ORACLE_STRATEGY_ATTEMPT_TIMEOUT_MS = 3000;
+export const GAS_ORACLE_STRATEGY_MAX_TIMEOUT_MS = 5000;
```

Comment:
With these values, you most likely won't be able to make more than one attempt. Also, the question is, is the `GAS_ORACLE_STRATEGY_MAX_TIMEOUT_MS` supposed to be for one gas price strategy but multiple attempts or across all the strategies?

Reply:
Good point :thinking: 

I was thinking per strategy attempt, but now that you mention it, I think it would make more sense to have the `MAX_TIMEOUT` be for all strategies combined as we wouldn't want the gas oracle to take too long from a single cycle, right?

I would need to refactor the `attemptGasOracleStrategy` a bit to make the `MAX_TIMEOUT` apply to all attempts. What do you think, though? Should these values also be configurable in the strategy?

We'd likely want each `ATTEMPT_TIMEOUT` be at least 1.5-2 seconds as often it takes that long for the provider to respond, especially for getting the blocks for `latestBlockPercentileGasPrice` strategy.

Reply:
I would say you're right and it would make sense to have a max timeout for all strategies combined.

Hmm, I'd say that we don't need these timeouts to be configurable at the moment. Just some sane defaults would imo suffice for the time being. Something like 10s for the global timeout across all the strategies (should be fine since the Airnode can run for up to 1 min anyway), something like 2s for an attempt, and 2 attempts for each call?